packages/publish-app-layer/src/index.tsx

```diff
@@ -45,6 +45,14 @@ export default function () {
   const onCrossClickCb = useCallback(() => {
     setShowPopup(false);
   }, []);
+
+  // keyboard shortcut for run and build
+  const handleKeyPress = useCallback((event: { repeat: any; ctrlKey: boolean; key: string; }) => {
+    if (!event.repeat && event.ctrlKey === true && event.key === "b")
```

Comment:
Please handle this for Mac as well. See this for an example. https://github.com/Atri-Labs/atrilabs-engine/blob/main/packages/app-template-layer/src/hooks/useListenCopyPaste.tsx#L19-L42

In Mac, command key is used instead of ctrl key. The command key is treated as meta key.

Reply:
```
  const handleKeyPress = useCallback((event:KeyboardEvent) => {
    const { repeat, ctrlKey, key,metaKey } = event;
    if (!repeat && ctrlKey === true && key === "b") {
      setShowPopup(!showPopup);
      callRunTaskApi();      
    }
    
    if (window.navigator && window.navigator.userAgent) {
      if (window.navigator.userAgent.indexOf("Mac")) {
        if (!repeat && metaKey === true && key === "b") {
          setShowPopup(!showPopup);
          callRunTaskApi();      
        }
      }
    }

  }, [callRunTaskApi, showPopup]);
```
Does this work @cruxcode ?

Reply:
This isn't right. The first if statement will work in Mac also. I don't want Mac user to be able to use "ctrl" key. They should only be able to use "command" key.

Reply:
Hey @cruxcode .Sorry I was not well for the past few days.So starting my work today.
 ```
const handleKeyPress = useCallback(
    (event: KeyboardEvent) => {
      const { repeat, ctrlKey, key, metaKey } = event;
      if (window.navigator && window.navigator.userAgent) {
        if (window.navigator.userAgent.indexOf("Mac") >= 0) {
          if (!repeat && metaKey === true && key === "b") {
            setShowPopup(!showPopup);
            callRunTaskApi();
          }
        } else if (!repeat && ctrlKey === true && key === "b") {
          if (!repeat && metaKey === true && key === "b") {
            setShowPopup(!showPopup);
            callRunTaskApi();
          }
        }
      }
    },
    [callRunTaskApi, showPopup]
  );
```